app/services/data_sync/elasticsearch/article.rb

```diff
@@ -0,0 +1,20 @@
+module DataSync
+  module Elasticsearch
+    class Article < Base
+      RELATED_DOCS = %i[
+        comments
+      ].freeze
+
+      SHARED_FIELDS = %i[
+        published
+        title
+      ].freeze
+
+      private
+
+      def comments
```

Comment:
I don't think you need this. I think the sync_related_documents will handle this fine
```ruby
def sync_related_documents
  self.class::RELATED_DOCS.each do |relation_name|
    __send__(relation_name).find_each(&:index_to_elasticsearch)
  end
end
```

Reply:
Just test it, it wouldn't work without it. If I comment it out and run it locally, I would get
```
NoMethodError:
  undefined method `comments' for #<DataSync::Elasticsearch::Article:0x000055ebbbca78b0>
```

If I understand this correctly, things listed in `RELATED_DOCS` is called as a method that I need to provide. Did I miss a step to make this work without that?

Reply:
AH, we used delegate! 
```
delegate :comments, to: :@updated_record
```

Reply:
Updated!